packages/components/src/Disclosure/Disclosure.tsx

```diff
@@ -54,26 +92,77 @@ export function Disclosure({
   const isBelowBreakpoint = exactWidth && exactWidth < Breakpoints.small;
   const isTitleString = typeof title === "string";
 
+  const containerClassNames = classnames(
+    styles.details,
+    UNSAFE_className.container,
+  );
+
+  const summaryClassNames = classnames(
```

Comment:
I left this as `summary` only because we also have `summaryWrap` and I thought it would be better to leave these names matching rather than rename 2 more elements in `UNSAFE`